webapp/src/index.tsx

```diff
@@ -7,7 +7,7 @@ import reducer from 'reducers';
 
 import {handleConnect, handleDisconnect, handleSubscriptionDeleted} from 'websocket';
 
-import {ChannelHeaderBtn} from 'components/buttons/action_buttons';
+import {ChannelHeaderButton} from 'components/buttons/channelHeaderButton';
```

Comment:
I see that some folder names are using snake case and some are using camel case. We should make it consistent.

Reply:
this may be happened due to multiple devs working on web app over the time and didn't look on the common practise